app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/dialogs/SendTabDialogWidget.java

```diff
@@ -58,6 +68,13 @@ protected void initialize(@NonNull Context aContext) {
         mAccounts = ((VRBrowserApplication)getContext().getApplicationContext()).getAccounts();
     }
 
+    @Override
+    public void releaseWidget() {
+        super.releaseWidget();
+
+        mSendTabDialog = null;
+    }
+
```

Comment:
Should this be called some where? Seems like we want to create a new one when we have a new activity?